app/models/update_existing_draft_edition.rb

```diff
@@ -79,4 +95,20 @@ def attributes
   def edition_attributes_from_payload
     payload.slice(*Edition::TOP_LEVEL_FIELDS)
```

Comment:
Yep, that wouldn't have worked anyway as you pointed out with the distinction between the payload fields and the database field names.